drivers/hmis/app/graphql/mutations/add_household_members_to_enrollment.rb

```diff
@@ -40,30 +36,29 @@ def resolve(household_id:, start_date:, household_members:)
 
         next enrollment if enrollment.present?
 
-        enrollment = Hmis::Hud::Enrollment.new(
+        Hmis::Hud::Enrollment.new(
```

Comment:
This isn't being saved or returned as far as I can tell am I missing something?

Reply:
maybe you still need
```suggestion
        enrollment = Hmis::Hud::Enrollment.new(
```

Reply:
won't it automatically return the last line, since it's a map function?

Reply:
ugh, yes, sorry the GitHub UI for this one was wonky.  Nothing to see here.